src/Pos/Launcher/Runner.hs

```diff
@@ -211,24 +242,44 @@ runServiceMode res bp@BaseParams{..} listeners outSpecs (ActionSpec action) = do
     usingLoggerName (lpRunnerTag bpLoggingParams) .
       runKademliaDHT (rmDHT res) .
       runPeerStateHolder stateM .
-      runServer (rmTransport res) listeners outSpecs . ActionSpec $
+      runServer_ (rmTransport res) listeners outSpecs . ActionSpec $
           \vI sa -> nodeStartMsg bp >> action vI sa
 
 runServer :: (MonadIO m, MonadMockable m, MonadFix m, WithLogger m, MonadDHT m)
   => Transport
   -> ListenersWithOut m
   -> OutSpecs
+  -> (Node m -> m t)
+  -> (t -> m ())
   -> ActionSpec m b
   -> m b
-runServer transport packedLS (OutSpecs wouts) (ActionSpec action) = do
+runServer transport packedLS (OutSpecs wouts) withNode afterNode (ActionSpec action) = do
     ourPeerId <- PeerId . getMeaningPart <$> currentNodeKey
     let (listeners', InSpecs ins, OutSpecs outs) = unpackLSpecs packedLS
         ourVerInfo = VerInfo protocolMagic lastKnownBlockVersion ins $ outs <> wouts
         listeners = listeners' ourVerInfo ++ protocolListeners
     stdGen <- liftIO newStdGen
     logInfo $ sformat ("Our verInfo "%build) ourVerInfo
     node (concrete transport) stdGen BiP (ourPeerId, ourVerInfo) $ \__node ->
-        pure $ NodeAction listeners (action ourVerInfo)
+        pure $ NodeAction listeners $ \sendActions -> do
+            st <- liftIO getMaskingState
+            logInfo $ sformat ("Node masking state is " % shown) st
```

Comment:
It's the async exception masking state. I'll remove this. Just had it here to verify that it's `Unmasked` at this point.